### ISSUE TITLE
asset push: policy now needs content type

### ISSUE THREAD

**Issue by wh0** [wh0/snail-cli#27]
https://support.glitch.com/t/cant-upload-an-obj-file/47202?u=wh0

spotted in a post on their support forum, it looks like requesting a policy now needs a content type specified?

we don't do that, so is our current code broken now?

**Comment by wh0** [comment]
confirmed broken

```
$ snail a push my-asset.png 
Error: Glitch projects policy response 400 not ok
    at Command.doAPush (/app/src/index.js:935:28)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by wh0** [comment]
fixed in v2.8.0
